system/liquid-ceph/templates/configmap.yaml

```diff
@@ -23,3 +23,10 @@ data:
 
     [client.admin]
     keyring = /etc/ceph-keyring/keyring
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: liquid-ceph-zone-mapping
+data:
+  zone-mapping.yaml: ""
```

Comment:
Why do you try to make the configmap name the thing that is configured? The standard approach is to take the zoneMapping config itself via `values.yaml` and render it out here (Ctrl-F `toYaml` on this repo for many many examples of how to do this). Then hardcode the configMap name at the usage site.